static/templates/search_list_item.hbs

```diff
@@ -0,0 +1,8 @@
+<div class="search_list_item">
+    {{{ description }}}
+    {{#if is_person}}
+    <div class="pill-container pill-container-btn">
+        {{> input_pill user_pill_context}}
+    </div>
+    {{/if}}
+</div>
```

Comment:
This is a preexisting issue that's hard to fix, but this HTML ordering isn't going to work well for languages with a different word order than English.  I'm not sure there's a good solution, so just noting it.

See https://zulip.readthedocs.io/en/latest/translating/internationalization.html for details on this problem.